### PR TITLE
Added docs for workflow_marked_places() function

### LINKED ISSUE #7785: [Workflow] Added workflow_marked_places() twig function
See https://github.com/symfony/symfony/pull/22180 and https://symfony.com/blog/new-in-symfony-3-3-workflow-improvements

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7785.

### CODE REVIEW

File: workflow/usage.rst [hunk 1]
```diff
@@ -249,9 +249,26 @@ This class has two more methods:
...
+
+``workflow_marked_places()``
+    Returns an array with the place names of the given marking.
+
+``workflow_has_marked_place()``
+    Returns ``true`` if the marking of the given object has the given state.
+
+.. versionadded:: 3.3
+    The ``workflow_marked_places()`` and ``workflow_has_marked_place()``
+    functions were added in Symfony 3.3.
```
Comment (xabbuh) [inline workflow/usage.rst]: [...] were introduced [...]

File: workflow/usage.rst [hunk 2]
```diff
@@ -249,9 +249,26 @@ This class has two more methods:
...
+
+The following example shows these methods in action:
```
Comment (xabbuh) [inline workflow/usage.rst]: functions

**Comment by xabbuh** [comment]
I think we can backport the listing to the 3.2 docs (omitting the two new Twig functions of course).

**Comment by HeahDude** [comment]
👍 for back porting 3.2 features.

File: workflow/usage.rst [hunk 3]
```diff
@@ -277,3 +294,6 @@ The links below will only be displayed when the action is allowed:
     {% if workflow_has_marked_place(post, 'to_review') %}
         <p>This post is ready for review.</p>
     {% endif %}
+
+    {# Get all the places related to the 'post' marking #}
+    {{ workflow_marked_places(post)|join(',') }}
```
Comment (javiereguiluz) [inline workflow/usage.rst]: This example is pretty stupid. Can you please suggest me an interesting and realistic example for `workflow_marked_places()`? Thanks!

**Comment by javiereguiluz** [comment]
Can you do the backporting while merging or should I make a new PR?

File: workflow/usage.rst (hunk 3 above)
Comment (HeahDude) [inline workflow/usage.rst]: What about something like:
```twig
{% if 'waiting_some_approval' in workflow_marked_places(post) %}
    <span class="special-label"></span>
{% endif %}
```
Many places may be needed for a "can", but some times, only one to display things.

Comment (javiereguiluz) [inline workflow/usage.rst]: Very nice! I added your example. Thanks!

**Comment by xabbuh** [comment]
I can do that while merging.

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**